packages/reports/addon/components/power-select-collection-options.ts

```diff
@@ -24,45 +25,23 @@ export type IndexedOptions<Option = ItemOptions[number]> = { option: Option; idx
 export default class PowerSelectCollectionOptions extends Options {
   declare args: Args;
 
-  /**
-   * @property {Number} maxDisplayedItems - max number of item to show at once
-   */
-  maxDisplayedItems = 6;
+  guid = guidFor(this);
```

Comment:
Is this being used?

Reply:
👍  good catch, the `id` was being overwritten by the `...attributes` so this is no longer needed